scripts/src/java/org/oppia/android/scripts/RepoFile.kt

```diff
@@ -0,0 +1,60 @@
+package org.oppia.android.scripts
+
+import java.io.File
+
+/** Helper class which contains all the file collection helper methods */
```

Comment:
```suggestion
/** Helper class for managing & accessing files within the project repository. */
```

I think this might be slightly clearer. Also, note the ending punctuation.

Reply:
Done.